dev-qt/qtnetwork/qtnetwork-5.14.1.ebuild

```diff
@@ -20,13 +20,18 @@ DEPEND="
 	libproxy? ( net-libs/libproxy )
 	networkmanager? ( ~dev-qt/qtdbus-${PV} )
 	sctp? ( kernel_linux? ( net-misc/lksctp-tools ) )
-	ssl? ( dev-libs/openssl:0=[bindist=] )
+	ssl? (
+		!libressl? ( dev-libs/openssl:0=[bindist=] )
+		libressl? ( dev-libs/libressl:0= )
+	)
 "
 RDEPEND="${DEPEND}
 	connman? ( net-misc/connman )
 	networkmanager? ( net-misc/networkmanager )
 "
 
+PATCHES=( "${FILESDIR}"/${P}-libressl.patch )
```

Comment:
missing bug reference, should have inline comment `# Not upstreamable`

Reply:
Thanks. Fixed.